src/components/CellViews/ImageCell.js

```diff
@@ -12,32 +12,36 @@ export const ImageCell = (column) => {
   const renderImagePreview = original.total_annotations === 0 || !root.showPreviews;
   const imgSrc = Array.isArray(value) ? value[0] : value;
 
-  return renderImagePreview ? (
-    <img
-      key={imgSrc}
-      src={imgSrc}
-      alt="Data"
-      style={{
-        maxHeight: "100%",
-        maxWidth: "100px",
-        objectFit: "contain",
-        borderRadius: 3,
-      }}
-    />
-  ) : (
-    <AnnotationPreview
-      task={original}
-      annotation={original.annotations[0]}
-      config={getRoot(original).SDK}
-      name={alias}
-      variant="120x120"
-      fallbackImage={value}
-      style={{
-        maxHeight: "100%",
-        maxWidth: "100px",
-        objectFit: "contain",
-        borderRadius: 3,
-      }}
-    />
-  );
+  return imgSrc ? (
```

Comment:
```suggestion
  if (!imgSrc) return null
```

might be a bit more readable